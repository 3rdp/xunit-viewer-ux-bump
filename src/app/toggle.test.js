import React from 'react'
import Toggle from './toggle'
import renderer from 'react-test-renderer'

test('renders toggle', () => {
  const tree = renderer.create(<Toggle onLabel='on' offLabel='off' />).toJSON()
  expect(tree).toMatchSnapshot()
})
