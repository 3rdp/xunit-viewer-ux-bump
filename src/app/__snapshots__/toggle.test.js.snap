// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders toggle 1`] = `
<button
  className="button toggle is-inactive "
  disabled={false}
  onClick={[Function]}
>
  <div
    className="toggle-rail"
  >
    <div
      className="toggle-handle"
    />
  </div>
  <span>
    off
  </span>
</button>
`;
